### PR TITLE
chore: Modify Connector section - README.md file

### LINKED ISSUE #281: data_connector: improve readme
**Is your feature request related to a problem? Please describe.**
The current readme for the data connector is very brief. 
We will improve in two aspects:
1. Making the image dynamic, like EDA module using gifs
2. Adding more descriptions, such as the strengths and benefits of data connector 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by juandavidospina**
# Description

Updating the "Connector" section in the README.md file. Three new animations were included (connector_main.gif, connector_pagination.gif, and connector_concurrency.gif) plus updates in descriptions and content. Closes #281.

# How Has This Been Tested?

Content verified using Markdown editor.

# Snapshots:

image

image

image

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have already squashed the commits and make the commit message conform to the project standard.
- [x] I have already marked the commit with "BREAKING CHANGE" or "Fixes #" if needed.
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,19 +102,47 @@ Don't forget to checkout the [examples] folder for detailed demonstration!
 
 ### Connector
 
-Connector provides a simple way to collect data from different websites, offering several benefits:
-* A unified API: you can fetch data using one or two lines of code to get data from many websites.
-* Auto Pagination: it automatically does the pagination for you so that you can specify the desired count of the returned results without even considering the count-per-request restriction from the API.
-* Smart API request strategy: it can issue API requests in parallel while respecting the rate limit policy.
+Connector provides a simple programming interface to collect structured data from different Web APIs (e.g., Twitter API, Yelp Fusion API, Spotify API, DBLP API), making web data collection easy and efficient, without requiring advanced programming skills.
```
Comment (peiwangdb) [inline README.md:105]: Change to the value proposition sentence.

**Review by dovahcrow — CHANGES_REQUESTED** [review]
I think there are still some changes from the last meeting?

**Comment by juandavidospina** [comment]
@dovahcrow Right. They are in progress. Target date: 2020-10-28.

**Comment by juandavidospina** [comment]
Replaced by #404.

**Pull request closed without merging**